test/evergreen.yml

```diff
@@ -2733,6 +2846,7 @@ buildvariants:
     posix_configure_flags: --enable-silent-rules --enable-diagnostic --enable-python --enable-zlib --enable-snappy --enable-strict --enable-static --prefix=$(pwd)/LOCAL_INSTALL
     python_binary: '/opt/mongodbtoolchain/v3/bin/python3'
     make_command: PATH=/opt/mongodbtoolchain/v3/bin:$PATH make
+    data-validation-stress-test-checkpoint-fp-hs-insert-common-params: -t r -W 3 -D -n 100000 -k 100000 -C cache_size=100MB
```

Comment:
Why did we chose to remove the `-p` flag here?

Reply:
> Why did we remove the `-p` flag here?

@luke-pearson This was the configuration we decided to proceed with. Do we need to include the `-p` option?

Reply:
@luke-pearson  Updated the configurations to include prepare, mixed-mode delete, and use timestamp flags.